apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V96__recreate_pre_adj_child_tables.sql

```diff
@@ -0,0 +1,197 @@
+-- Rebuild the schema to match new RDA specifications and truncate tables to end soft launch
+
+drop table rda.fiss_audit_trails;
+drop table rda.fiss_diagnosis_codes;
+drop table rda.fiss_payers;
+drop table rda.fiss_proc_codes;
+drop table rda.mcs_adjustments;
+drop table rda.mcs_audits;
+drop table rda.mcs_details;
+drop table rda.mcs_diagnosis_codes;
+drop table rda.mcs_locations;
+truncate table rda.fiss_claims;
+truncate table rda.mcs_claims;
+truncate table rda.message_errors;
+truncate table rda.rda_api_progress;
+-- Not truncating the metadata or MBI tables
+alter table rda.claim_message_meta_data ${logic.alter-rename-column} received_date ${logic.rename-to} last_updated;
+
```

Comment:
Maybe we should rename the index on received_date.  It's currently `claim_message_meta_data_received_date_idx`.